apps/src/javalab/Javalab.js

```diff
@@ -191,7 +193,7 @@ Javalab.prototype.init = function(config) {
     isSubmitted: !!config.level.submitted
   });
 
-  registerReducers({javalab});
+  registerReducers({javalab, javalabConsole});
```

Comment:
I'm guessing we couldn't put this under javalab (like `javalab.console`) without changing how registerReducers works right?